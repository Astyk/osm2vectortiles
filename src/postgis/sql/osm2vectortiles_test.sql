CREATE EXTENSION osm2vectortiles;
SELECT point_to_tiles(1, 1, 14);
