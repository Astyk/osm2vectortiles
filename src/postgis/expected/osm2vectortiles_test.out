CREATE EXTENSION osm2vectortiles;
SELECT point_to_tiles(1, 1);
 point_to_tiles 
----------------
 (0,0,0)
 (1,0,1)
 (2,1,2)
 (4,3,3)
 (8,7,4)
 (16,15,5)
 (32,31,6)
 (64,63,7)
 (128,127,8)
 (257,254,9)
 (514,509,10)
 (1029,1018,11)
 (2059,2036,12)
 (4118,4073,13)
 (8237,8146,14)
(15 rows)

